p2p/conn/connection.go

```diff
@@ -632,8 +632,8 @@ FOR_LOOP:
 				// never block
 			}
 		case *tmp2p.Packet_PacketPong:
-			c.latency = time.Since(<-c.pingTimeCh).Milliseconds() / 2
-			c.pongTS = time.Now().UnixMilli()
+			c.pongTS = time.Now().UTC().UnixMilli()
```

Comment:
What exactly was the issue with the previous implementation? The output is a time difference so it doesn't matter whether both times are UTC or local.

BTW, what about using [tmtime.Now()](https://github.com/tendermint/tendermint/blob/main/types/time/time.go#L9-L11) which already returns time in UTC?
```suggestion
			c.pongTS = tmtime.Now().UnixMilli()
```